src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Truck is starting its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck has stopped working.");
```

Comment:
Similarly, the stop message should also be more informative. Consider changing it to 'Truck has stopped its work.' to clearly indicate the type of machine.